psalm-baseline.xml

```diff
@@ -49,28 +49,6 @@
       <code>null</code>
     </PossiblyNullPropertyAssignmentValue>
   </file>
-  <file src="Packages/Framework/Neos.Cache/Classes/Backend/RedisBackend.php">
```

Comment:
Hmm, still wonder why your psalm always removes those errors...

Reply:
Because I have the Redis extension, so it's not undefined?

Reply:
Let's try with annotations… :)

Reply:
Let's try with annotations on class instead of property level…

Reply:
Oh, 'cmon, psalm. That sucks.